### DISCUSSION TITLE
Lock token does not match existing lock

### DISCUSSION THREAD

**Discussion by sicenul** [abraunegg/onedrive#2419] [General]
I use onedrive using docker compose. But since for a reasong I have to reinstall all docker things, I forgot to rerun onedrive image for almost a month (I'm not sure) :smiley: 

Today I rerun the image, and found this error:
```
onedrive  | ERROR: Microsoft OneDrive API returned an error with the following message:
onedrive  |   Error Message:    HTTP request returned status code 423 (Locked)
onedrive  |   Error Reason:     Lock token does not match existing lock
onedrive  |   Error Timestamp:  2023-06-15T04:30:56
onedrive  |   API Request ID:   6250***deleted***e15c
```

How do I get the token and input it to docker container, to unlocked and remove the error message?
Thanks.

Notes: I'm using driveone/onedrive:2.4-debian image

**Comment by abraunegg** [comment]
@sicenul 
It is best to review the following: https://github.com/OneDrive/onedrive-api-docs/issues/912

This is an API issue, not a problem with the client. This is not something that this client can solve.

**Comment by sicenul** [comment]
Ok thank you, will check on that.

**Comment by jehillert** [comment]
I use 1 drive as a typical user, just for keeping personal files on the cloud.  I just got this error when I tried to drag and drop a microsoft word document into the browser.  That is just tip of the iceberg, last few days it seems a slew of new syncing issues are destroying any new work I do.

**Comment by abraunegg** [comment]
@jehillert 

> I use 1 drive as a typical user, just for keeping personal files on the cloud. I just got this error when I tried to drag and drop a microsoft word document into the browser. 

Can you provide a little more of detailed walk through *how* you are triggering this type of event?

If this is *repeatable* it can be investigated further and tested against - and, if this is a bug, then fixed. Right now, I have *never* seen this error message myself despite detailed usage and running the client + developing & testing it extensively. 

> That is just tip of the iceberg, last few days it seems a slew of new syncing issues are destroying any new work I do.

OK .. this is more concerning. Start with the basics .. what application version are you using - the output of `onedrive --version` is needed. Next what distribution are you using? Next, what is the output of `onedrive --display-config` showing? Lastly what is your account type - Personal, Business or a SharePoint Site?

A sync issue should *not* be destroying your work at all. If there are sync conflicts - a backup of the file is made.

Can you give more details around your workflow so your workflow can be replicated to try and identify how or where any data loss is occuring.

**Comment by jehillert** [comment]
Don't kill me, I didn't notice this was a client for Linux.  I just saw the repo was called "onedrive".  That makes sense why you responded.  If it had been microsoft, they wouldn't have.  Sorry about that.
